### PR TITLE
add secondary button

### PR DESCRIPTION
**Pull request by panentheos**
This adds a formal "secondary button" with the proper font styles and a transparent background. We're in the process of adding more of these.

### CODE REVIEW

File: src/components/common.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const buttonPrimary = function(props, children) {
       style: {
         ...Style.elements.button,
         border: 'none', borderRadius: 5, color: 'white',
-        height: '2.25rem', paddingLeft: '1.5rem', paddingRight: '1.5rem',
+        height: '2.25rem', padding: '0 1.5rem 0 1.5rem',
```
Comment (MattPutnam) [inline src/components/common.js]: `padding: '0 1.5rem'`?

**Review by MattPutnam — APPROVED** [review]

File: src/components/common.js (hunk 1 above)
Comment (panentheos) [reply src/components/common.js]: Yep

**Review by zarsky-broad — APPROVED** [review]
Only thought: unless we expect a bunch of hover styles, there's no reason to use an interactive.

**Comment by panentheos** [comment]
Doesn't Interactive also provide some accessibility features by default?

**Pull request merged**